test/index.js

```diff
@@ -59,13 +59,6 @@ test('shows a repo', async t => {
   t.true(agoRegExp.test(await browser.getText('time.updated-at')));
 });
 
-test('checks that the games load up', async () => {
```

Comment:
removing these should have been another commit in the same PR, as this is related to the fact that Ryan removed the game , and is unrelated to the test runner problems.  Also note that it wasnt mentioned in the commit message.  Anyway, this is minor and it is very distinct changes within the same commit.  

I am guessing Ryan will appreciate this being sent upstream (as two commits, and probably best left to coding phase 2), and it should almost apply without much difficulty, and likewise try the same with Phil, but that is very optional as that codebase probably has other problems which are not related to your project.

Reply:
Btw removing this is in the commit message ;)
`Remove game functionality test code in test/index.js.`